service/templates/admin.html

```diff
@@ -1840,6 +1840,10 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 										<label>Persistent DNS Domain Name</label>
 										<input type="text" ng-model="Settings.providers.aws.persistent_dns.domain">
 									</md-input-container>
+									<md-input-container class="control" style="width:45%;">
+										<label>Generated JSON Files S3 Prefix</label>
+										<input type="text" ng-model="Settings.providers.aws.parser_project.generated_json_prefix">
+									</md-input-container>
 									<md-input-container class="control" style="width:45%;">
```

Comment:
The rest of these input forms have to have their `style` updated so that it's still in an alternating pattern of `width:45%; margin-left:50px;` -> `width:45%`. Otherwise, the admin settings page formatting gets messed up.

Reply:
oops I missed that but updated a screen shot on description